pkg/dbutil/common.go

```diff
@@ -648,14 +650,14 @@ func ExecSQLWithRetry(ctx context.Context, db *sql.DB, sql string, args ...inter
 		_, err = db.ExecContext(ctx, sql, args...)
 		takeDuration := time.Since(startTime)
 		if takeDuration > SlowWarnLog {
-			log.Warn("exec sql slow", zap.String("sql", sql), zap.Reflect("args", args), zap.Duration("take", takeDuration))
+			log.Debug("exec sql slow", zap.String("sql", sql), zap.Reflect("args", args), zap.Duration("take", takeDuration))
```

Comment:
No `Warn` for `SlowWarnLog` now, how about giving `SlowWarnLog` another name?

Reply:
update in 758aca3